build/build-sdk-images/cpp/build.sh

```diff
@@ -17,4 +17,4 @@
 set -ex
 
 cd ./sdks/cpp
-make build install archive VERSION=$VERSION
\ No newline at end of file
+make build verify install archive VERSION=$VERSION
```

Comment:
Should verify be part of a new test.sh step?
Context: https://github.com/googleforgames/agones/tree/master/build/build-sdk-images

WDYT?

Reply:
As someone who was trying to run sdk tests yesterday, I think it's better as part of the build step. I think if a test step just runs formatting checks and not any actual tests, it'll make people _think_ that tests are running when in fact they aren't.

Reply:
Devil's advocate.

If we had unit tests, would you want verify to run in the same test.sh target as the unit tests?

Reply:
If we had tests I'd be fine with it in either the build or test target. You could argue the build target is still the right place, and you always have to build before testing anyway, so even if you just ran the test target you'd validate formatting. Or you could argue that checking formatting is a 'test' against the correctness of the code's layout, so it should be a test. 

But, right now we don't have tests, so I think it should be in the build target.

Reply:
Sounds good to me - just wanted to talk it through :+1: 

I'm going to approve this!